cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperator.java

```diff
@@ -174,4 +180,12 @@ private void reconcileAll(String trigger) {
         return result;
     }
 
+    public static String secretName(String cluster) {
+        return cluster + CERTS_SUFFIX;
+    }
+
+    public static Secret generateSecret(ClusterCa clusterCa, String namespace, String cluster) {
+        Secret secret = clusterCa.clusterOperatorSecret();
+        return SecretGenerator.generateSecret(clusterCa, secret, namespace, ClusterOperator.secretName(cluster), "cluster-operator", Labels.fromString(""), null);
```

Comment:
The intention of `KafkaResources` is to allow people who what to programmatically manipulate a Strimzi cluster with helpful methods they might need, and in particular to avoid them having to embed knowledge of how we name the resources they have a legitimate interest in. They might legitimately need to know about the Zookeeper StatefulSet (even though we don't support people using that zookeeper for their own purposes, they might want to know about it for knowing when it's is ready, as we do ourselves in the tests). Similarly with the other Secrets they need to import the cluster CA cert into their pods for adding to the trust store. But I can't think of why they would need to know about this Secret. So I'm inclined to say (at least for now) that it's fine to keep this method private to the CO.